doc/man7/provider-base.pod

```diff
@@ -607,6 +607,78 @@ or maximum. A -1 indicates that the group should not be used in that protocol.
 
 =back
 
+=head3 "TLS-SIGALG" Capability
+
+The "TLS-SIGALG" capability can be queried by libssl to discover the list of
+TLS signature algorithms that a provider can support. Each signature supported
+can be used for client- or server-authentication in addition to the build-in
```

Comment:
`built-in` not `build-in`

Reply:
so changed in https://github.com/openssl/openssl/pull/19312/commits/e5582173f0db49a28edf0907db053ea66e10d3dc

Reply:
This seems not to be fixed after all?

Reply:
In this case, what's not done is s/build-in/built-in/

Reply:
now done.